### ISSUE TITLE
GUI not displaying forwards

### ISSUE THREAD

**Issue by microlancer** [cryptosharks131/lndg#358]
My /forwards page is blank. 

I inspected the page with Chrome Dev Tools and saw:

```
Uncaught (in promise) TypeError: Cannot read properties of null (reading 'default')
    at Object.chan_in_alias (forwards:24:58)
    at Object.render (helpers.js:54:41)
    at build_routed (forwards:122:50)
    at HTMLDocument.<anonymous> (forwards:104:5)
```
![Screenshot from 2023-12-22 15-55-16](https://github.com/cryptosharks131/lndg/assets/869094/4199be34-02da-4bc7-bd19-39b62d229e89)

In the network tab, I see the fetch http://localhost:8889/api/forwards/?limit=50&format=json and it is returning a successful list of JSON containing all the forwards.

So the data is there, but the JS is failing and so the page is blank.

Any ideas?

**Comment by microlancer** [comment]
Here is some more added context. The line where it's crashing is this missing value.

![Screenshot from 2023-12-22 16-00-32](https://github.com/cryptosharks131/lndg/assets/869094/9c4b4768-6265-44f2-84b8-953449a52253)

`chan_in_alias` or `chan_out_alias` are sometimes blank

**Comment by microlancer** [comment]
From what I can tell, they are blank when the channel was closed and no longer exists and LND can't look up the name anymore.

**Comment by microlancer** [comment]
If anyone is having issues, it's a pretty easy fix, just update these two lines:

```
      "chan_in_alias": f => ({innerHTML: f.chan_in_alias != null ? f.chan_in_alias.default(f.chan_id_in) : ''}),
      "chan_out_alias": f => ({innerHTML: f.chan_out_alias != null ? f.chan_out_alias.default(f.chan_id_out) : ''}),
```
And the page should load forwards again.

Also, a good thing to keep in mind is that the forwards are loaded slowly from LND over time and many background job requests, so they might start off showing only very old forwards.

A good UI improvement would be to have some indicator that the data loading is still in progress.

**Comment by cryptosharks131** [comment]
Please try the fix that was deployed in the v1.8.0 branch.
`"chan_in_alias": f => ({innerHTML: f.chan_in_alias || f.chan_id_in}),`
`"chan_out_alias": f => ({innerHTML: f.chan_out_alias || f.chan_id_out}),`

**Comment by microlancer** [comment]
> Please try the fix that was deployed in the v1.8.0 branch. `"chan_in_alias": f => ({innerHTML: f.chan_in_alias || f.chan_id_in}),` `"chan_out_alias": f => ({innerHTML: f.chan_out_alias || f.chan_id_out}),`

It works for me!
